src/applications/simple-forms/shared/components/IntroductionPageView.jsx

```diff
@@ -47,11 +47,21 @@ export const IntroductionPageView = ({
       </SaveInProgressIntro>
       {additionalChildContent || null}
       <p />
-      <va-omb-info
-        res-burden={resBurden}
-        omb-number={ombNumber}
-        exp-date={expDate}
-      />
+      {!customPrivacyActStmt ? (
```

Comment:
This re-surfaces va-omb-info's support for custom Privacy-Act-Statement modal-content.